com.engine.Acestream.json

```diff
@@ -0,0 +1,150 @@
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "cp -p /usr/share/automake-*/config.{sub,guess} ."
+                    ]
+                }
+            ],
+            "modules": [
+                {
+                    "name": "python2-pycairo",
+                    "buildsystem": "meson",
+                    "config-opts": ["-Dpython=python2"],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://github.com/pygobject/pycairo/releases/download/v1.18.1/pycairo-1.18.1.tar.gz",
+                            "sha256": "70172e58b6bad7572a3518c26729b074acdde15e6fee6cbab6d3528ad552b786"
+                        }
+                    ]
+                },
+                {
+                    "name": "pygobject",
+                    "config-opts": ["--disable-glibtest"],
+                    "cleanup": ["/bin"],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://download.gnome.org/sources/pygobject/2.28/pygobject-2.28.7.tar.xz",
+                            "sha256": "bb9d25a3442ca7511385a7c01b057492095c263784ef31231ffe589d83a96a5a"
+                        }
+                    ]
+                }
+            ]
+        },
+            "libs/libappindicator/libappindicator-gtk2-12.10.json",   
+         {
+            "name": "acestream",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm755 apply_extra /app/bin/apply_extra",
+                "install -Dm755 acestream.sh /app/bin/acestream",
+                "for size in 16 24 32 48 64 128 256; do\n                    rsvg-convert -w $size -h $size -f png -o $size.png logo-small.svg\n                    install -Dm644 $size.png /app/share/icons/hicolor/${size}x${size}/apps/com.engine.Acestream.png\n                done",
+                "install -Dm644 logo-small.svg export/share/icons/hicolor/scalable/apps/com.engine.Acestream.svg",
+                "install -Dm644 com.engine.Acestream.appdata.xml /app/share/appdata/com.engine.Acestream.appdata.xml",
+                "install -Dm644 com.engine.Acestream.desktop /app/share/applications/com.engine.Acestream.desktop"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "apply_extra",
+                    "commands": [
+                        "tar -zxvf acestream.tar.gz",
+                        "rm -f acestream.tar.gz"
+                    ]
+                },
+               {
+                    "type": "script",
+                    "dest-filename": "acestream.sh",
+                    "commands": [
+                        "exec /app/extra/start-engine --client-gtk --log-file $HOME/.ACEStream/acestream.log \"$@\""
```

Comment:
Why are you logging stuff at $HOME?

Reply:
By default, the application saves the logs to / tmp, which requires permissions, so there is an alternate path to .ACEStream (the cache from the application is saved in this folder). Yes, it's a crutch, but an application doesn't need to be granted elevated privileges just because it wants to save logs where it doesn't need to.

Reply:
Please use the xdg-cache dir for that.

Reply:
can you help with the implementation? I'm just stuck.

Reply:
Done. Thanks to Barthalion for the advice.